client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/useClinicalDetails.ts

```diff
@@ -198,13 +196,11 @@ const useClinicalDetails = (parameters: useClinicalDetailsIncome): useClinicalDe
 
     const getSymptomsList = (patientInvestigation: any) => {
         const symptoms: string[] = patientInvestigation.investigatedPatientSymptomsByInvestigationId.nodes.map((symptom: any) => symptom.symptomName);
-        if (patientInvestigation.otherSymptomsMoreInfo) symptoms.push(otherSymptomFieldName);
```

Comment:
Why were these lines removed?

Reply:
Because other is already in the db, so there is no need in adding it